<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="report_label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>12</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Report</string>
    </property>
   </widget>
   <widget class="QLabel" name="filter_label">
    <property name="geometry">
     <rect>
      <x>38</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="price_checkbox">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>215</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Price Limit Range</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_high">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>215</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter High Price</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price_low">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>215</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Low Price</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return_btn">
    <property name="geometry">
     <rect>
      <x>699</x>
      <y>256</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Return MainPage</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calender_start">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>344</width>
      <height>195</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="start_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QCalendarWidget" name="calender_end">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>344</width>
      <height>195</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="end_label">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>196</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>End</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="income_checkbox">
    <property name="geometry">
     <rect>
      <x>39</x>
      <y>62</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Income</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cost_checkbox">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>62</y>
      <width>83</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Cost</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="yesterday_radio">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>100</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Yesterday</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lastweek_radio">
    <property name="geometry">
     <rect>
      <x>143</x>
      <y>100</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LastWeek</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="lastmonth_radio">
    <property name="geometry">
     <rect>
      <x>252</x>
      <y>101</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>LastMonth</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="last3month_radio">
    <property name="geometry">
     <rect>
      <x>362</x>
      <y>101</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Last 3 Month</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="custom_period_radio">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>101</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom Period</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="type_checkbox">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>261</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Money Type</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="resource_checkbox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>262</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Resource Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="type_lineedit">
    <property name="geometry">
     <rect>
      <x>189</x>
      <y>259</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Money Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="resource_lineedit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>258</y>
      <width>181</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Enter Money Resource</string>
    </property>
   </widget>
   <widget class="QPushButton" name="report_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>310</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Get Report</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
